src/views/SpaceDelegates.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+import { ExtendedSpace } from '@/helpers/interfaces';
+import { useInfiniteScroll } from '@vueuse/core';
+
+const {
```

Comment:
Same here composables should go after props.

I know its a draft, just some comments.